第4回課題（READ処理）

tshima2 opened the pull request:
第4回課題（READ処理）の実装です。
GoogleMapAPIで地図ベースの画面にしてみました。
![無題](https://user-images.githubusercontent.com/76857613/151666916-b8752134-dd1a-4ed0-8129-f99016d4219b.png)

（概要）
- dbのadapterにmysql2spatialを使用, 位置情報を持つSiteモデル作成
- モデルスペックのひな形作成
- jQueryインストールと設定
- Site一覧と参照機能作成

tshima2 committed:
- dbのadapterにmysql2spatialを使用, 位置情報を持つSiteモデル作成
- jQueryインストールと設定
- モデルスペックのひな形作成
- Site一覧(地図入り）, 参照機能作成

yuta-ushijima commented on app/controllers/sites_controller.rb:
```diff
@@ -0,0 +1,33 @@
+
+    #output site location(json) for googlemap
+    _points = []
+    @sites.each do |site|
```
繰り返し文を使いつつ新しい配列を作成する際は、each + pushよりもmapを使う方がメモリ効率などの点からもベターですね！
参考: https://qiita.com/its/items/4be63e69cf29c8ea2676

yuta-ushijima reviewed (approved):
課題的にはLGTMです！1点補足でコメントします！

tshima2 committed:
DBパスワード環境変数名の変更

tshima2 committed:
Site新規作成時の確認機能追加

tshima2 committed:
SiteモデルのCRUD処理を実装
・ Show/Editアクション追加
・ 一覧画面にて地図上から各アクション起動UIを追加
・ 各アクションのView用にCSS, アイコン（font-awesome）を追加

tshima2 committed:
- jQueryUI削除, font-awesome追加

tshima2 committed:
exclude local rails server startup-script

tshima2 commented:
指摘事項の修正反映し, SiteのCRUD処理をひとおとり実装
